guides/plugins/themes/theme-inheritance-configuration.md

```diff
@@ -0,0 +1,121 @@
+# Theme inheritance configuration
+
+## Overview
+
+This guide explains how you can use a theme as a basic corporate design theme and create inherited themes for 
+special purposes like holiday time or a sales week.
+
+Imagine you have a theme which is applying your corporate design to the storefront. With your colors, your logo and 
+your icons. But on a special week in the year you have additional requirements for a special design, like a discount 
```

Comment:
Maybe we should not mention icons here because right now icons cannot really be handled via theme.json. Maybe "and you configuration fields"